src/libs/ReportUtils.js

```diff
@@ -2200,29 +2200,29 @@ function canLeaveRoom(report, isPolicyMember) {
 }
 
 /**
- * @param {string[]} participants
+ * @param {number[]} participantAccountIDs
  * @returns {Boolean}
  */
-function isCurrentUserTheOnlyParticipant(participants) {
-    return participants && participants.length === 1 && participants[0] === sessionEmail;
+function isCurrentUserTheOnlyParticipant(participantAccountIDs) {
+    return participantAccountIDs && participantAccountIDs.length === 1 && participantAccountIDs[0] === sessionAccountID;
```

Comment:
FYI it looooks like `sessionAccountID` and `currentUserAccountID` both refer to SESSION.accountID (idk why my name is on the blame there 😬 ) so we couldddd clean that up in this PR or a separate one

Reply:
Similar to `sessionEmail` vs `currentUserEmail`

Reply:
Updated!